mb.pid

```diff
@@ -0,0 +1 @@
+85932
```

Comment:
Remove this file, and if you want you could add `mb.pid` and `mb.log` to the `.gitignore`.